src/applications/post-911-gib-status/containers/IntroPage.jsx

```diff
@@ -80,12 +80,30 @@ export class IntroPage extends React.Component {
   }
 
   render() {
-    const content = this.getContent();
-
     return (
       <div>
         <h1>Post-9/11 GI Bill Statement of Benefits</h1>
-        {content}
+        <div>
+          <AlertBox
+            headline={downtimeSoonAlertHeadline}
+            content={downtimeSoonAlertContent}
+            isVisible={
+              this.props.uptimeRemaining &&
+              this.props.uptimeRemaining <= DOWNTIME_SOON_CUTOFF
+            }
+            status="warning"
+          />
+          <AlertBox
+            headline={systemUpAlertHeadline}
+            content={systemUpAlertContent}
+            isVisible={
+              !this.props.uptimeRemaining ||
+              this.props.uptimeRemaining > DOWNTIME_SOON_CUTOFF
+            }
+            status="success"
+          />
```

Comment:
We don't need to call `getContent` anymore?

Reply:
Oh gosh I'm so happy you caught this, thank you!!